README.md

```diff
@@ -41,33 +41,37 @@ The `calico-node` service is a worker that configures the network endpoints for
 We provide a command line tool, `calicoctl`, which makes it easy to configure and start the Calico services listed above, and allows you to interact with the etcd datastore to define and apply network and security policy to the containers you create.
 
 ```
+Override the host:port of the ETCD server by setting the environment variable
+ETCD_AUTHORITY [default: 127.0.0.1:4001]
+
 Usage:
-  calicoctl node --ip=<IP> [--node-image=<DOCKER_IMAGE_NAME>] [--ip6=<IP6>] [--force-unix-socket]
+  calicoctl node --ip=<IP> [--node-image=<DOCKER_IMAGE_NAME>] [--ip6=<IP6>]
   calicoctl node stop [--force]
   calicoctl status
   calicoctl shownodes [--detailed]
   calicoctl profile show [--detailed]
-  calicoctl profile add <PROFILE>
-  calicoctl profile remove <PROFILE>
+  calicoctl profile (add|remove) <PROFILE>
   calicoctl profile <PROFILE> tag show
-  calicoctl profile <PROFILE> tag add <TAG>
-  calicoctl profile <PROFILE> tag remove <TAG>
+  calicoctl profile <PROFILE> tag (add|remove) <TAG>
   calicoctl profile <PROFILE> rule show
   calicoctl profile <PROFILE> rule json
   calicoctl profile <PROFILE> rule update
   calicoctl profile <PROFILE> member add <CONTAINER>
-  calicoctl ipv4 pool add <CIDR>
-  calicoctl ipv4 pool del <CIDR>
-  calicoctl ipv4 pool show
-  calicoctl ipv6 pool add <CIDR>
-  calicoctl ipv6 pool del <CIDR>
-  calicoctl ipv6 pool show
-  calicoctl container add <CONTAINER> <IP>
+  calicoctl (ipv4|ipv6) pool (add|remove) <CIDR>
+  calicoctl (ipv4|ipv6) pool show
+  calicoctl (ipv4|ipv6) bgppeer rr (add|remove) <IP>
+  calicoctl (ipv4|ipv6) bgppeer rr show
+  calicoctl (ipv4|ipv6) container <CONTAINER> (add|remove) <IP> [--interface=<INTERFACE>]

```

Comment:
I feel like for this command we can drop the (ipv4|ipv6) option and infer it from the address itself.

Actually, this is probably true for all the commands.